Dependencies/xr/Include/XR.h

```diff
@@ -120,6 +111,63 @@ namespace xr
         Identifier ID{};
     };
 
+    struct FieldOfView
+    {
+        float AngleLeft;
+        float AngleRight;
+        float AngleUp;
+        float AngleDown;
+    };
+
+    enum class DetectionBoundaryType
+    {
+        Box,
+        Frustum,
+        Sphere
+    };
+
+    struct Frustum
+    {
+        Pose Pose{};
+        FieldOfView FOV{};
+        float FarDistance;
+    };
+
+    struct DetectionBoundary
+    {
+        DetectionBoundaryType Type{ DetectionBoundaryType::Sphere };
```

Comment:
nit: if only sphere is used, do we want to name it accordingly?

Reply:
sphere is the default value used here. The detection boundary type is specified through the Babylon.js api